src/components/OsCveTable/OsCveTable.tsx

```diff
@@ -0,0 +1,313 @@
+import React, { useEffect, useState, useMemo } from "react";
+import { Table, ConfigProvider, theme } from "antd";
+import styles from "./OsCveTable.module.scss";
+import Search from "../Technologies/Search";
+import { useColorMode } from "@docusaurus/theme-common";
+import type { ColumnsType } from "antd/es/table";
+import Admonition from "@theme/Admonition";
+import path from "path";
+import { Link } from "react-router-dom";
+
+type OsCve = {
+  uid: string;
+  displayName: string;
+  osDistribution: string;
+  osVersion: string;
+  k8sDistribution: string;
+  k8sVersion: string;
+  timeLastUpdated: string;
+  hash?: string;
+};
+
+interface ProviderCveItem {
+  kind: string;
+  metadata: {
+    uid: string;
+    summary: string;
+    advCreatedTimestamp: string;
+    advLastModifiedTimestamp: string;
+  };
+  spec: {
+    assessment: {
+      impact: string;
+      severity: string;
+      thirdParty: {
+        isDependentOnThirdParty: boolean;
+      };
+    };
+    impact: {
+      isImpacting: boolean;
+      impactedVersions: string[];
+    };
+  };
+}
+
+interface AllCVEList {
+  provider: ProviderCveItem[];
+}
+
+const columns: ColumnsType<OsCve> = [
```

Comment:
The columns look great, but should we display the CVE ID somewhere as well?

For example, `CVE-2022-1234`. I'm assuming we get given this info, if not, then please ignore 😄 .

Reply:
From reading the `os-example.json`, is this because the OS vulnerabilities may contain multiple CVE IDs within the `linkedVulnerabilities` section?

Reply:
Yes, each page here is a summary of all CVEs related to an OS image. That's why there is no CVE IDs on here. There are like hundreds of them per image 😅